README.md

```diff
@@ -61,8 +59,8 @@ These are all available inputs.
 ## Regional holidays
 
 This tool supports blocking merges based on regional holidays. For example, February 11th is "National Foundation Day"
-in Japan, and you can block merges on such a holiday via `prohibited-days-dates` with `H:Japan`. In addition, you can
-also block on the day before the regional holiday with `BH:Japan`.
+in Japan, you can block merges on such a holiday via `prohibited-days-dates` with `H:Japan`. In addition, you can
+also block the day before the regional holiday with `BH:Japan`.
```

Comment:
Thank you. I misunderstood "block" can be used as an intransitive verb, but it should be followed by any object. How about this one? In this context, "block" should be followed by "merges."
 
```suggestion
also block merges on the day before the regional holiday with `BH:Japan`.
```